### ISSUE TITLE
Rapid Application Publish fails with Page Extensions / Views

### ISSUE THREAD

**Issue by fvet** [microsoft/AL#5594]
I've added some views to an existing page extension ...

![image](https://user-images.githubusercontent.com/25268332/72079524-60c62580-32fb-11ea-93bb-e4eb37fda059.png)

... published via **Rapid** Publishing, but got not the expected result (only one view got added).

![image](https://user-images.githubusercontent.com/25268332/72078691-de893180-32f9-11ea-8ad0-6e68d5a6e823.png)

Only after full publishing, all views became available.

![image](https://user-images.githubusercontent.com/25268332/72078703-e21cb880-32f9-11ea-8d30-d74802555e4b.png)

Can you please improve the 'Rapid Publishing' feature and ensure that all mods on page extensions are published / available after 'Rapid Publishing' (instead of forcing us to use a full publish which degrades the current dev experience a lot)

![image](https://user-images.githubusercontent.com/25268332/72079546-6a4f8d80-32fb-11ea-8115-96824efaa968.png)

**Comment by kalberes** [comment]
RAD should publish all changed objects. If it does not then it is a bug. In your case was there only page extension.  Have you noticed anything on teh srever (eventviewer execptions, sql issues)
A simple repro would help a lot. Else we are guessing and those issues where we have to guess will not get much attention.

**Comment by atoader** [comment]
This issue has been closed because it needs more information and has not had recent activity. Please open a new issue if you believe this has not been resolved and reference the current issue.
